template/module/demo/assets.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright <YEAR(S)> <AUTHOR(S)>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
```

Comment:
Not sure when is the moment to update that... I think many people still develop for v8. Maybe when v10 comes out we can update this template more safely.